exetera/core/fields.py

```diff
@@ -2922,11 +3096,36 @@ def data(self):
         self._ensure_valid()
         if self._value_wrapper is None:
             if self._write_enabled:
-                self._value_wrapper = WriteableFieldArray(self._field, 'values')
+                self._value_wrapper = WriteableFieldArray(self, 'values')
             else:
-                self._value_wrapper = ReadOnlyFieldArray(self._field, 'values')
+                self._value_wrapper = ReadOnlyFieldArray(self, 'values')
         return self._value_wrapper
 
+    def attach(self, view):
+        self._view_refs.append(view)
+
+    def detach(self, view=None):
```

Comment:
This detach is actually `notify_deletion`. This is a standard part of subject_observer, but so is `detach`, which is for the observer to detach from a given subject